superset-frontend/src/views/CRUD/data/dataset/AddDataset/DatasetPanel/DatasetPanel.tsx

```diff
@@ -123,6 +128,24 @@ const StyledTable = styled(Table)`
   right: 0;
 `;
 
+const StyledAlert = styled(Alert)`
```

Comment:
Related to coment on line 20, we shuld repelase use of supertThem with:
`const StyledAlert = styled(Alert)(
  ({ theme }) => `

and replace all the `supersetTheme` with `theme`
for example:
`border: 1px solid ${theme.colors.info.base};

Reply:
FIxed in [`this commit`](https://github.com/apache/superset/pull/22136/commits/86549660bd4d6ac53e0b7723ab3e9370e37d2a28).